scripts/project.js

```diff
@@ -0,0 +1,24 @@
+window.addEventListener("DOMContentLoaded", function () {
+    const changeStateForms = document.getElementsByClassName("frm_chg_stt");
+
+    function confirmSubmit(event) {
+        if (this.dataset.confirmationQuestion && !confirm(this.dataset.confirmationQuestion)) {
+            event.preventDefault();
+        }
+    }
+
+    for (const changeStateForm of changeStateForms) {
+        changeStateForm.addEventListener("submit", confirmSubmit);
+    }
```

Comment:
Can we group the variables with the loops/ifs -- I think this makes it easier to read.
```suggestion
    function confirmSubmit(event) {
        if (this.dataset.confirmationQuestion && !confirm(this.dataset.confirmationQuestion)) {
            event.preventDefault();
        }
    }

    const changeStateForms = document.getElementsByClassName("frm_chg_stt");
    for (const changeStateForm of changeStateForms) {
        changeStateForm.addEventListener("submit", confirmSubmit);
    }
```

Ditto the block below with `newTopicLink`.

Reply:
Done this.